pkg/config/setup/config.go

```diff
@@ -1767,6 +1767,7 @@ func ResolveSecrets(config pkgconfigmodel.Config, secretResolver secrets.Compone
 		config.GetInt("secret_refresh_interval"),
 		config.GetBool("secret_backend_command_allow_group_exec_perm"),
 		config.GetBool("secret_backend_remove_trailing_line_break"),
+		filepath.Join(config.GetString("run_path"), "refresh-audit.json"),
```

Comment:
For now we use the term `reload` when talking with customer about this feature. And we have another file in there called `version-history.json` to track agent version across upgrades/downgrades. I think we should keep the same nomenclature.

```suggestion
		filepath.Join(config.GetString("run_path"), "secret-reload-history.json"),
```

Also, should we keep the file name in the secret comp and take the `run_path` as a parameter ?